pandas/_libs/lib.pyx

```diff
@@ -286,7 +287,7 @@ def fast_unique_multiple(list arrays, sort: bool = True):
         try:
             uniques.sort()
         except TypeError:
-            # TODO: RuntimeWarning?
+            warnings.warn("The values in the array are unorderable", RuntimeWarning)
```

Comment:
Can you indicate somethig about what to do about this? (eg passing the `sort=False` keyword? I don't know if that is possible for all cases that may end up here)